hbase-server/src/test/java/org/apache/hadoop/hbase/master/procedure/TestWALProcedureStoreOnHDFS.java

```diff
@@ -156,7 +158,7 @@ public void abortProcess() {}
       thread[i].start();
     }
 
-    Thread.sleep(1000);
+    Thread.sleep(2000);
```

Comment:
Increasing sleep time by 1 more sec will help for all env? Or we are planning to commit this first and then observe branch test results for a while?